.woodpecker/.test-pr.yml

```diff
@@ -3,27 +3,25 @@ steps:
     image: danlynn/ember-cli:5.8.0-node_20.12
     commands:
       - npm ci
-  build:
-    image: danlynn/ember-cli:5.8.0-node_20.12
-    commands:
-      - npm run build
```

Comment:
This was moved to the `test-build` workflow. This should speed things up since it was now holding up the lint and test steps..